datasetinsights/commands/evaluate.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+import click
+
+import datasetinsights.constants as const
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    help="Start model evaluation tasks for a pre-trained model.",
+    context_settings=const.CONTEXT_SETTINGS,
+)
+@click.option(
+    "-c",
+    "--config",
+    type=click.STRING,
+    required=True,
+    help="Path to the config estimator yaml file.",
+)
+@click.option(
+    "-p",
+    "--checkpoint-file",
+    type=click.STRING,
+    required=True,
+    help="URI to a checkpoint file.",
+)
+@click.option(
+    "-d",
+    "--data-root",
+    type=click.STRING,
+    default=const.DEFAULT_DATA_ROOT,
+    help="Root directory on localhost where datasets are located.",
+)
+@click.option(
+    "-l",
+    "--tb-log-dir",
+    type=click.STRING,
+    default=const.DEFAULT_TENSORBOARD_LOG_DIR,
+    help=(
+        "Path to the directory where tensorboard events should be stored. "
+        "This Path can be GCS URI (e.g. gs://<bucket>/runs) or full path "
+        "to a local directory."
+    ),
+)
+@click.option(
+    "-w",
+    "--workers",
+    type=click.INT,
+    default=0,
+    help=(
+        "Number of multiprocessing workers for loading datasets. "
+        "Set this argument to 0 will disable multiprocessing which is "
+        "recommended when running inside a docker container."
+    ),
+)
+@click.option(
+    "--kfp-metrics-dir",
+    type=click.STRING,
+    default=const.DEFAULT_KFP_METRICS_DIR,
+    help="Path to the directory where Kubeflow Metrics files are stored.",
```

Comment:
Is this local only or will a GCS URI also work? If its a local file only it makes sense to make the type `click.Path(exists=True)`

Reply:
This will more likely only support local directory. I will make this `click.Path`.